platform/linux-generic/arch/default/odp_wait_until.h

```diff
@@ -0,0 +1,49 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2023 Nokia
+ */
+
+#ifndef ODP_DEFAULT_WAIT_UNTIL_H_
+#define ODP_DEFAULT_WAIT_UNTIL_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <odp/api/cpu.h>
+
+#include <stdint.h>
+
+static inline void _odp_sevl(void)
+{
+	/* empty */
+}
+
+static inline int _odp_wfe(void)
+{
+	return 1;
+}
+
+#define _odp_monitor_u8(addr, mo) __atomic_load_n((addr), (mo))
+#define _odp_monitor_u32(addr, mo) __atomic_load_n((addr), (mo))
+#define _odp_monitor_u64(addr, mo) __atomic_load_n((addr), (mo))
+#define _odp_monitor_u128(addr, mo) __atomic_load_n((addr), (mo))
+
+static inline void _odp_wait_until_not_equal_u32(uint32_t *val, uint32_t expected)
```

Comment:
Ah, I had a brain fart when naming these functions. I ended up renaming all the new wait until functions in V3  to avoid conflict with the existing `_odp_wait_until_equal_acq_u32()` function. I didn't want to touch the scalable scheduler implementation and longer term plan is to remove it anyway.